src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import ThemeSwitch from '../../components/ThemeSwitch';
+import { contactsMap } from '../../constants';
+
+import mednetLogo from '../../assets/images/mednet-logo.png';
+
+import './Sidebar.css';
+
+class Sidebar extends React.Component {
+  state = {
+    isSidebarOpened: false,
+  };
+
+  sidebarOnCloseHandler = () => {
+    this.setState({ isSidebarOpened: false });
+  };
+
+  toggleSidebar = () => {
+    const { isSidebarOpened } = this.state;
+    this.setState({ isSidebarOpened: !isSidebarOpened });
+  };
+
+  getContactItem = ({ name, link, to }) => {
+    return (
+      <li key={`contacts-contact-${name}`} className="sidebar__contact-item">
+        <a
+          href={`${to}${link}`}
+          target="_blank"
+          rel="noopener noreferrer"
+          className="sidebar__contact-link"
+        >
+          <FormattedMessage {...messages[name]} />
+        </a>
+      </li>
+    );
+  };
+
+  render() {
+    const { isSidebarOpened } = this.state;
+    let sidebarContent = '';
```

Comment:
Try never to use `let`. And if possible, use `const`. Never mutate or redefine data, but create a new object instead.

so do:
```
renderSomeComponent = () => ...
render() {
  return (
    <div>
      {isSidebarOpened ? (this.renderSomeComponent()) : (this.renderOtherComponent)}
```

Reply:
fixed